src/octopeer-github/main/MainController.ts

```diff
@@ -71,17 +63,21 @@ class MainController {
      * @param tab   the Tab to check for
      */
     private testAndSend(tab: Tab) {
```

Comment:
Could you maybe include this in the TSDoc of the function?
On May 19, 2016 4:00 PM, "Robin van Heukelum" <notifications@github.com>
wrote:

> In src/octopeer-github/main/MainController.ts
> <https://github.com/thervh70/ContextProject_RDD/pull/59#discussion_r63883108>
> :
>
> > @@ -71,17 +63,21 @@ class MainController {
> >       * @param tab   the Tab to check for
> >       */
> >      private testAndSend(tab: Tab) {
>
> This function wasn't written by me, ask @mpsijm
> <https://github.com/mpsijm>
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/thervh70/ContextProject_RDD/pull/59/files/92e8e35b3ffe9df66766d0ca1c1b24e585c0fe0b#r63883108>
>